workflow/defaults/places.yaml

```diff
@@ -18,9 +18,11 @@ default_places: &default_places
   SOCA_STATIC: !expand "{GODAS_RC}/soca-static/{doc.da_settings.NI}x{doc.da_settings.NJ}x{doc.da_settings.NK}"
   SOCA_CONFIG:   !expand "{ROOT_GODAS_DIR}/src/soca-bundle/soca-config/emc"             # soca-config path
   DCOM_ROOT: !expand "{GODAS_RC}/ocean_observations/data"         # Observation root path for raw obs
-  IODA_ROOT: !expand "{GODAS_RC}/ioda_observations"               # Processed observations
+  IODA_ROOT: !expand "{GODAS_RC}/ioda_observations"   # Processed observations
   MOD_PATH: !expand "{ROOT_GODAS_DIR}/modulefiles"
   MODEL_SCRATCH: "/scratch2/NCEPDEV/ocean/Guillaume.Vernieres/mom6-cice5-fv3/{NI}x{NJ}x{NK}"
   SHORT_TERM_TEMP: !calc doc.platform.short_term_temp             # short term storage
   LONG_TERM_TEMP:  !calc doc.platform.long_term_temp              # long term storage
   RUNDIR: !expand "{LONG_TERM_TEMP}/{tools.env('USER')}/rundir/{doc.names.experiment}" # RUN directory
+  SOCA_ANALYSIS:  "/scratch2/NCEPDEV/marineda/Shastri.Paturi/Shastri.Paturi/rundir/workflowtest_QCexp_SST25"   # SOCA ANALYSIS For HOFX
```

Comment:
@ShastriPaturi Help me here, this points to your path, how different is it from the RUNDIR?

Reply:
@flampouris it is different from RUNDIR, since it will be the one in which the present HoFX is running and the my path, at the moment, is the RUNDATE for a completed 1 month cycle.

Reply:
@flampouris, we need to talk in person to resolve the location of the SOCA_ANALYSIS path.
layout/hofx_only.yaml, cases/hofx3d.yaml are not the right places for the same.

Reply:
@flampouris after a careful thought, I have removed the SOCA_ANALYSIS variable from default/place.yaml and added it to cases/hofx3d.yaml. This seems to be the appropriate place for it for the time-being, at least. We can brain storm with @guillaumevernieres and @JianKuang-NOAA, if necessary as an update to this PR. 
If you agree, then I will push the changes.